tests/api/test_session_view_service.py

```
@@ -1127,11 +1160,72 @@ def load_session(self, session_id: str):
 
+def test_session_view_skips_recovery_substitute_from_view_load() -> None:
+    corrupt = _session("session_corrupt", "Corrupt", "2026-02-01T00:00:00")
+    recovery = _session("recovery_20260608_185439", "Recovery", "2026-02-02T00:00:00")
+
+    class _RecoverySubstituteManager(_Manager):
+        def __init__(self) -> None:
+            super().__init__([corrupt])
+            self.sessions.clear()
+            self.current_session = None
+
+        def load_session(self, session_id: str):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `load_session`

(ANN202)